### ISSUE TITLE
ganeti man page referrs to design documents

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#977]
Originally reported of Google Code with ID 921.

```

...more precisely to design-oob.rst and design-resource-model.rst.

This should be changed, as typical Ganeti setup might install
the man pages but not the design documents.
```
Originally added on 2014-08-21 15:10:35 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 6d02596fb9ea929df657cdc2d2a7f12417ca3fed
Author: Klaus Aehlig <aehlig@google.com>
Date:   Thu Sep 25 13:49:39 2014 +0200

    Remove references to design documents in man pages
    
    Ganeti often gets installed with man pages but without design
    documents. This makes sense, as design documents aim at Gaenti
    developers, while man pages are also for persons just using
    Ganeti. Therefore, this patch removes references to draft design
    documents. For the other references,
    - it adds a bit more explanation so that the man page is self-contained,
    - reduces the reference to the name of the document, and
    - adds the Ganeti version where it was implemented, so that it
      easier to find on docs.ganeti.org.
    
    Fixes issue #921.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

```
Originally added on 2014-09-25 13:13:13 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:02 +0000 UTC.

Changed State: Released
